formats/calendar/SRF_Calendar.php

```diff
@@ -357,7 +367,12 @@ function displayCalendar( $events ) {
 		}
 
 		$cur_month = self::intToMonth( $cur_month_num );
-		$cur_year = date( 'Y', time() );
+
+		if ( is_numeric( $this->mStartYear ) && ( intval( $this->mStartYear ) == $this->mStartYear ) ) {
+			$cur_year = $this->mStartYear;
+		} else {
+			$cur_year = date( 'Y' );
```

Comment:
:+1: 

I haven't checked it, I just wanted to make sure it doens't return with something unexpected.